VideoLocker/src/main/java/org/edx/mobile/model/course/CourseComponent.java

```diff
@@ -25,7 +26,7 @@
     private BlockCount blockCount;
     protected CourseComponent parent;
     protected CourseComponent root;
-    protected List<CourseComponent> children = new ArrayList<>();
+    protected List<CourseComponent> children;
```

Comment:
Can we make these private, since there's an accessor for them anyway? That way we can feel a little safer aboutwhich class is responsible for them not being null.